angular/demo/src/app/samples/modal/playground.route.ts

```diff
@@ -10,7 +10,8 @@ const undefinedConfig = getUndefinedValues(getModalDefaultConfig());
 	standalone: true,
 	imports: [AgnosUIAngularModule],
 	providers: provideHashConfig('modal'),
-	template: `<au-modal #widget></au-modal>`,
+	selector: 'app-playground-modal',
```

Comment:
As mentioned in #89 , please do not put a selector in the default component of `a .route.ts file`.